### DISCUSSION TITLE
'float' object is not subscriptable

### DISCUSSION THREAD

**Discussion by shintaroudlotulhanafia** [AI4Finance-Foundation/FinRL#698] [Q&A]
I try to remake this one, https://github.com/AI4Finance-Foundation/FinRL/blob/322e51225e0b626ae0526da4c78c1203293b8b68/tutorials/4-Optimization/FinRL_HyperparameterTuning_using_Optuna_basic.ipynb But I got error warning "'float' object is not subscriptable" at cell below
![optuna](https://user-images.githubusercontent.com/107638433/185719634-8dc0d583-9ab6-4be6-8db7-f80021978e4b.jpg)

**Comment by forestgithub123** [comment]
I have encountered the same problem. Have you solved it?
